src/Framework/Sdk/ReferencedSdk.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Build.Framework
+{
+    /// <summary>
+    ///     Represents a software development kit (SDK) that is referenced in a &lt;Project /&gt; or &lt;Import /&gt; element.
+    /// </summary>
+    public sealed class ReferencedSdk : IEquatable<ReferencedSdk>
+    {
+        /// <summary>
+        ///     Initializes a new instance of the ReferencedSdk class.
+        /// </summary>
+        /// <param name="name">The name of the SDK.</param>
+        /// <param name="version">The version of the SDK.</param>
+        public ReferencedSdk(string name, string version)
+        {
+            Name = name;
+            Version = version;
+        }
+
+        /// <summary>
+        ///     Gets the name of the SDK.
+        /// </summary>
+        public string Name { get; }
+
+        /// <summary>
+        ///     Gets the version of the SDK.
+        /// </summary>
+        public string Version { get; }
+
+        /// <summary>
+        /// </summary>
+        /// <param name="other"></param>
+        /// <returns></returns>
+        public bool Equals(ReferencedSdk other)
+        {
+            if (ReferenceEquals(null, other))
+                return false;
+            if (ReferenceEquals(this, other))
+                return true;
+            return string.Equals(Name, other.Name) && Equals(Version, other.Version);
```

Comment:
We decided to be case-sensitive for Sdk names? I _think_ we did as a side effect of the initial implementation on case-sensitive file systems, but I don't remember for sure. Should be doc'ed either way.